draft-ietf-tls-dtls-rrc.md

```diff
@@ -160,6 +173,48 @@ threshold) and initiate the return routability check that proceeds as follows:
 After this point, any pending send operation is resumed to the bound peer
 address.
 
+{{path-challenge-reqs}} and {{path-response-reqs}} contain the requirments for
+the initiator and responder roles, broken down per protocol phase.
+
+## Path Challenge Requirements {#path-challenge-reqs}
+
+* The initiator MAY send multiple `return_routability_check` message of type
+  path_challenge to cater for packet loss on the probed path.
+  * Each path_challenge SHOULD go into different transport packets.
+  * Each path_challenge MUST have different unpredictable data.
+* The initiator MAY use padding using the record padding mechanism available in
+  DTLS 1.3, and 1.2 when CID is enabled on its sending direction) up to the
+  anti-amplification limit to probe if the PMTU for the new path is still
+  acceptable.
+
+## Path Response Requirements {#path-response-reqs}
+
+* The responder MUST NOT delay sending an elicited path_response message.
+* The responder MUST send exactly one path_response messages for each received
+  path_request.
+* The responder MUST send the path_response on the network path where the
+  corresponding path_challenge has been received, so that validation succeeds
+  only if the path is functional in both directions.
+  * The initiator MUST NOT enforce this behaviour
+* The responder MAY use padding using the record padding mechanism available in
+  DTLS 1.3, and 1.2 when CID is enabled on its sending direction) up to the
+  anti-amplification limit to probe if the PMTU for the new path is still
+  acceptable.
+
+## Timer Choice {#timer-choice}
+
+When setting T, implementations are cautioned that the new path could have a
+longer round-trip time than the original.
+
+In settings where there is external information about the RTT, implementations
+SHOULD use [1.5,3]xRTT estimate as T.
+<cref>TODO pick a number in the interval</cref>
+
+If an implementation has no way to obtain information regarding the current
+RTT, a value of 1s SHOULD be used.  If there is a-priori knowledge of the
```

Comment:
The "default recommendation" 1s from RFC6347 is already adapted for some upper layer definitions (or experience). I would add to chose the "default recommendation" of the use-case, or 1s, if no other assumption is made. E.g. my experience for NB-IoT is about 2.5s.

Reply:
I hope I have captured that sentiment in the sentence that follows:
_"If there is a-priori knowledge of the network environment (e.g., endpoints deployed in constrained networks) a different, more suitable value MAY be chosen."_
which should cater for what you noted without naming names.

Reply:
Hm, yes, I stopped read at the "1s SHOULD be used". So, sure, the second sentence covers my cases.
Just as proposal:

" If there is a-priori knowledge of the network environment (e.g., endpoints deployed in constrained networks) a different, more suitable value SHOULD be chosen.

If an implementation has no way to obtain information regarding the current
RTT, and has no a-priori knowledge of the network environment, a value of 1s, according RFC6347, 4.2.4.1  SHOULD be used."

Otherwise, maybe too many will chose the 1s.

Reply:
I don't think a SHOULD without a precise value would work.  I have created https://github.com/thomas-fossati/draft-tls13-iot/issues/18 to deal with this in the IOT-PROFILE draft and the corresponding text in 
c31472f.

Reply:
For me that depends on what should be achieved. 

If using 1s is considered a good default value, then the text proposed in this PR will be ok. 
Please note, that for the other message exchanges in DTLS 1.2, [RFC6347 4.2.4.1](https://datatracker.ietf.org/doc/html/rfc6347#section-4.2.4.1) that (short) time of 1s is amended by "double the value at each retransmission, up to no less than the RFC 6298 maximum of 60 seconds."
That helps, for slower paths, even if it may be considered to be not that optimal. 
But for RCC, I don't see a definition, which relaxes that 1s and so using *only* 1s by default, without defining, how that is relaxed, will in my opinion fail on all cases with slower paths.

Therefore, I would still prefer, to have the "a-priori knowledge" listed before that default of 1s.

> I don't think a SHOULD without a precise value would work.

I think, a precise `1s` will not work in too many cases.